pkg/system/phase4_configuring.go

```diff
@@ -526,6 +526,25 @@ func (r *Reconciler) setDesiredEndpointMounts(podSpec *corev1.PodSpec, container
 		util.MergeVolumeMountList(&container.VolumeMounts, &secretVolumeMounts)
 	}
 
+	if r.NooBaa.Spec.BucketLogging.LoggingType == nbv1.BucketLoggingTypeGuaranteed &&
+		util.KubeCheck(r.BucketLoggingPVC) {
+		bucketLogVolumes := []corev1.Volume{{
+			Name: "bucket-logging-volume",
+			VolumeSource: corev1.VolumeSource{
+				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+					ClaimName: r.BucketLoggingPVC.Name,
+				},
+			},
+		}}
+		util.MergeVolumeList(&podSpec.Volumes, &bucketLogVolumes)
+
+		bucketLogVolumeMounts := []corev1.VolumeMount{{
+			Name:      "bucket-logging-volume",
+			MountPath: "/etc/logs/bucket-logs",
```

Comment:
let's move this outside to some constant in the reconciler, as we will need to send this to core and endpoints as an env variable, so he will know he needs to work with persistent logging and not the old one.

Reply:
let's also move the mount name to a shared constant

Reply:
Done.